src/Common/src/Interop/OSX/Interop.CoreFoundation.cs

```diff
@@ -86,6 +104,25 @@ internal static SafeCreateHandle CFStringCreateWithCString(IntPtr utf8str)
             return CFStringCreateWithCString(IntPtr.Zero, utf8str, CFStringBuiltInEncodings.kCFStringEncodingUTF8);
         }
 
+        /// <summary>
+        /// Creates a CFStringRef from a span of chars.
+        /// Follows the "Create Rule" where if you create it, you delete it.
+        /// </summary>
+        /// <param name="source">The chars to make a CFString version of.</param>
+        /// <returns>A CFStringRef on success, otherwise a SafeCreateHandle(IntPtr.Zero).</returns>
+        internal static unsafe SafeCreateHandle CFStringCreateFromSpan(ReadOnlySpan<char> source)
+        {
+            fixed (char* sourcePtr = source)
+            {
+                return CFStringCreateWithBytes(
+                    IntPtr.Zero,
+                    (IntPtr)sourcePtr,
+                    new CFIndex(source.Length * 2),
```

Comment:
Why *2?  It's in bytes?

Reply:
Yeah, `CFStringCreateWithBytes`'s input is bytes, our input is chars.